.changeset/wicked-zoos-look.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-graph': patch
+'@finos/legend-studio': patch
+---
+
+Add feature for editing filter in Mapping Editor
```

Comment:
So, this does not make sense, please split this into 2 changesets. One with empty message and one with `@finos/legend-studio` with the message `Add form editor for Pure Instance set implementation filter`